### PR TITLE
tests: adds mage target for CRS benchmarks.

### PR DESCRIPTION
**Pull request by jcchavezs**
Ease benchmark running

### CODE REVIEW

**Review by M4tteoP — APPROVED** [review]
lgtm

File: testing/coreruleset/coreruleset_test.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func BenchmarkCRSCompilation(b *testing.B) {
 	}
 }
 
-func BenchmarkCRSSimpleGET(b *testing.B) {
+func BenchmarkCRSRuntimeSimpleGET(b *testing.B) {
```
Comment (anuraaga) [inline testing/coreruleset/coreruleset_test.go:56]: I'm not too clear on what Runtime means here. If it means running time is the operation, I think all benchmarks measure that so doesn't need to be in name

**Review by anuraaga — APPROVED** [review]

**Comment by fzipi** [comment]
@jcchavezs Care to take a look at comments, or just merge?

**Comment by fzipi** [comment]
@jcchavezs Ping.

**Pull request closed without merging**